### PR TITLE
Added icecast module

### PR DESCRIPTION
**Pull request by k0ral**

### CODE REVIEW

File: nixos/modules/module-list.nix [hunk 1]
```diff
@@ -91,8 +91,9 @@
   ./services/amqp/activemq/default.nix
   ./services/amqp/rabbitmq.nix
   ./services/audio/alsa.nix
-  # Disabled as fuppes it does no longer builds.
```
Comment (iElectric) [inline nixos/modules/module-list.nix]: singular `builds`

Comment (k0ral) [inline nixos/modules/module-list.nix]: Something wrong with `fuppes no longer builds` ?

File: nixos/modules/services/audio/icecast.nix [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    systemd.services.icecast = {
+      after = [ "network.target" ];
+      description = "Icecast Network Audio Streaming Server";
+      wantedBy = [ "multi-user.target" ];
+
+      preStart = "mkdir -p ${cfg.logDir} && chown ${cfg.user}:${cfg.group} ${cfg.logDir}";
+      serviceConfig = {
+        Type = "simple";
+        ExecStart = "${pkgs.icecast}/bin/icecast -c ${configFile}";
+        ExecReload = "/usr/bin/kill -HUP $MAINPID";
```
Comment (lethalman) [inline nixos/modules/services/audio/icecast.nix]: /usr/bin/?

Comment (k0ral) [inline nixos/modules/services/audio/icecast.nix]: Oops... Fixed.

**Comment by lethalman** [comment]
Haven't tested but looks fine, thanks.

**Pull request merged**